teachertool/src/services/localStorage.ts

```diff
@@ -0,0 +1,48 @@
+import { ErrorCode } from "../types/errorCode";
+import { logError } from "./loggingService";
+
+const KEY_PREFIX = "teachertool";
+const AUTORUN_KEY = [KEY_PREFIX, "autorun"].join("/");
+
+function getValue(key: string, defaultValue?: string): string | undefined {
+    return localStorage.getItem(key) || defaultValue;
```

Comment:
Huh. Didn't know this existed. Wonder if I should switch the indexedDb meatdata table over to localstorage instead...

Reply:
Capacity of local storage is severely limited. We should only use it to store small small things like strings and bools. In fact, I should remove the APIs for reading/writing JSON objects.

Reply:
Yeah rubrics themselves should be in indexedDb, but the "lastActiveRubric" is just a key value pair to the name of the most recent rubric, which I think would be a good fit for this? Currently it's in its own metadata table in the indexedDb. I was originally going to say we could put autoRun in the same metadata table, but I think it actually makes more sense for me to move lastActiveRubric(Name) to localstorage.

Reply:
That makes sense.